### PR TITLE
Fix for t&c links

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Fixes issues mentioned in 
https://github.com/cowprotocol/cowswap/issues/373 and https://github.com/cowprotocol/cowswap/issues/875

### CODE REVIEW

File: src/custom/components/WalletModal/index.tsx [hunk 1]
```diff
@@ -12,12 +14,24 @@ const TermsWrapper = styled.div`
...
+  const toggleWalletModal = useWalletModalToggle()
+
   return (
     <TermsWrapper>
       <Trans>
         By connecting a wallet, you agree to GnosisDAO&apos;s{' '}
-        <ExternalLink href={Routes.TERMS_CONDITIONS}>Terms &amp; Conditions</ExternalLink> and acknowledge that you have
-        read, understood, and agree to them.{' '}
+        <ExternalLink
+          href="#"
```
Comment (anxolin) [inline src/custom/components/WalletModal/index.tsx:25]: nit: why not using just the regular URL here so it opens in new tab. It just feels easier, and user can read in the other tab and don't loose the context of the wallet.

Anyways, this also works for me

**Review by anxolin — COMMENTED** [review]
Please, before merging keep an eye on #legal channel and make sure they agree with whatever changes we do. Remind them if it takes a lot of time

**Comment by elena-zh** [comment]
Hey @nenadV91 , changes LGTM!
However, as @anxolin mentions above, it would be great to open the link in a new tab (as it currently works on Prod). 

Thanks

**Review by elena-zh — APPROVED** [review]
LGTM now!

File: src/custom/components/WalletModal/index.tsx (hunk 1 above)
Comment (shoom3301) [reply src/custom/components/WalletModal/index.tsx:20]:
```
<Link to={Routes.TERMS_CONDITIONS}>Terms &amp; Conditions</Link>
```
The Link from `'react-router-dom` looks more suitable here (with styled)

**Comment by anxolin** [comment]
@nenadV91 what should we do with this one?

**Comment by anxolin** [comment]
@nenadV91 merge?

**Comment by nenadV91** [comment]
As mentioned by legal, we will merge these changes for now, and once the new ToC is created we will use that one.

**Pull request merged**